akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -166,7 +166,7 @@ abstract class ActorRef extends java.lang.Comparable[ActorRef] with Serializable
  * There are implicit conversions in package.scala
  * from ActorRef -&gt; ScalaActorRef and back
  */
-trait ScalaActorRef { ref: ActorRef =>
+trait ScalaActorRef { ref: ActorRef with InternalActorRef with ActorRefScope =>
```

Comment:
is this change safe?

Reply:
Yes. Because in old code:
```scala
ScalaActorRef { _: ActorRef => ..}
ActorRef { _: InternalActorRef => ..}
InternalActorRef { _: ActorRefScope => ..}
```
a `ScalaActorRef` must be an `InternalActorRef` and `ActorRefScope` already.
dotty only ensure this by a compile error